source/simulator/solver.cc

```diff
@@ -764,7 +764,19 @@ namespace aspect
   Simulator<dim>::solve_stokes ()
   {
     TimerOutput::Scope timer (computing_timer, "Solve Stokes system");
-    pcout << "   Solving Stokes system... " << std::flush;
+
+    const std::string name = [&]() -> std::string
+    {
+      if (parameters.stokes_solver_type == Parameters<dim>::StokesSolverType::block_gmg)
+        return "GMG";
+      if (parameters.use_direct_stokes_solver)
+        return "direct";
+      if (parameters.use_bfbt)
+        return "BFBT-AMG";
```

Comment:
I think I would prefer "AMG-BFBT", because it is a modification of the AMG solver, which is the more important aspect. But I do not feel strongly about it, let me know what you think.

Reply:
Sure, I don't mind.